src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -103,6 +103,7 @@ angular
  * @param {boolean=} md-autoselect If set to true, the first item will be automatically selected
  *     in the dropdown upon open.
  * @param {string=} md-menu-class This will be applied to the dropdown menu for styling
+ * @param {string=} md-input-class This will be applied to the input for styling
```

Comment:
Can you please add a note that this attribute is only valid when a `md-floating-label` is defined?

Reply:
How is this:

> This will be applied to the input for styling. This attribute is only valid when a `md-floating-label` is defined?